go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.0
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0 // indirect
+	github.com/lib/pq v1.1.1 // indirect
```

Comment:
in case you were wondering why like I was
```
go: downloading github.com/lib/pq v1.1.1
# github.com/lib/pq
github.com/pomerium/pomerium/pkg/storage/redis
github.com/pomerium/pomerium/pkg/storage/redis.test
github.com/ory/dockertest/v3
github.com/ory/dockertest/v3.test
github.com/lib/pq

```